### ISSUE TITLE
Predefined height and Parameters

### ISSUE THREAD

**Issue by cherimo** [HeyAlex/BottomDrawer#9]
Hey There,

Nice Library, my compliments for your work.

I have more 2 questions then bugs.
Is it possible to give it a predefined height when you open the bottom drawer? Now it always goes half the screen first.
And my second question is it possible to give it some parameters with it when you open the bottom drawer?
So that for example I can show a Profile fragment.

Thank you in advance.

**Comment by HeyAlex** [comment]
Hi!

Yes, it's always opens to half of a screen. We probably need to make an api for half ratio to let change this behavior.

Need to make an api to configure default state [here](https://github.com/HeyAlex/BottomDrawer/blob/00eadc5f115a88e2d85e2a359496e9b203e4d6fe/bottomdrawer/src/main/java/com/github/heyalex/utils/BottomDrawerDelegate.kt#L164)

I can try to look on this, but can't promise when it happens. It would be nice if someone can make a PR.

**Comment by forceporquillo** [comment]
Hi Alex! 

I'd like to contribute and add some new features. Can you assign this to me? I'll PR asap once this request approved! Thanks!

PS. I'm using this library too. Thanks for your work!

**Comment by Srt9930** [comment]
Yes, this issue is valid when the BottomDrawer has no child or says that the recyclerView is empty then it collapsed and not open even half. we just can see the toolbar.

**Comment by Srt9930** [comment]
ahanhan... I take my words back! The problem was occurring because the child elements have a height of wrap_content, After replacing them with match_parant the problem gets solved.
